jobs/JGDAS_ENKF_DIAG

```diff
@@ -22,31 +22,33 @@ GDATE=$(${NDATE} -${assim_freq} ${CDATE})
 gPDY=$(echo ${GDATE} | cut -c1-8)
 gcyc=$(echo ${GDATE} | cut -c9-10)
 GDUMP=${GDUMP:-"gdas"}
+GDUMP_ENS=${GDUMP_ENS:-"enkf${GDUMP}"}
 
 export CASE=${CASE_ENKF}
-export CDUMP_OBS=${CDUMP_OBS:-${CDUMP}}
+export CDUMP_OBS=${CDUMP_OBS:-${CDUMP/enkf}}
+export CDUMP_CTL=${CDUMP_CTL:-${CDUMP/enkf}}
```

Comment:
What is intent of introducing `CDUMP_CTL`?

Reply:
Same for `CDUMP_OBS`

Reply:
`CDUMP_OBS` was a pre-existing variable. I created `CDUMP_CTL` so as not to misuse `CDUMP_OBS` for the deterministic member (even though they will be the same for now). I think this all goes away in the COM refactor when I revert `CDUMP` and use it per our discussion for both of these.